packages/components/src/molecules/CheckboxField/CheckboxField.tsx

```diff
@@ -28,7 +27,7 @@ const CheckboxField = forwardRef<HTMLDivElement, CheckboxFieldProps>(
           id={id}
           value={value ?? label}
           name={name}
-          defaultChecked={checked}
+          defaultChecked={otherProps.checked}
```

Comment:
**question**: What it really changes? It will still get the `checked` value from props 🤔

Reply:
The difference is that now _checked_ is no longer destructured. Before, checked was only passed to `defaulChecked`, and therefore, it was not possible to control the state of checked. Since we removed the destructuring, to maintain the previous behavior, we access checked through otherProps, which contains all the properties and attributes of the checkbox.

Reply:
Ok, so let's just add an optional chaining to avoid the `undefined` value of this prop

```suggestion
          defaultChecked={otherProps?.checked}
```